File: CS/Q212620/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.Xpo;
using DevExpress.Xpo.DB;
using Northwind;

namespace Q212620 {
    public partial class Form1 : Form {
        public Form1() {
            XpoDefault.ConnectionString = AccessConnectionProvider.GetConnectionString(@"..\..\nwind.mdb");
            InitializeComponent();
            XPVirtualTreeListDataBase data = new XPVirtualTreeListDataBase(session1);
            data.SetChildren(new XPCollection(session1, typeof(Categories)));
            treeList1.DataSource = data;
        }
    }
}

File: VB/Q212620/Form1.vb
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.Xpo
Imports DevExpress.Xpo.DB
Imports Northwind

Namespace Q212620

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            XpoDefault.ConnectionString = AccessConnectionProvider.GetConnectionString("..\..\nwind.mdb")
            InitializeComponent()
            Dim data As XPVirtualTreeListDataBase = New XPVirtualTreeListDataBase(session1)
            data.SetChildren(New XPCollection(session1, GetType(Categories)))
            treeList1.DataSource = data
        End Sub
    End Class
End Namespace
